### PR TITLE
Add code for Cubes of N numbers in JavaScript.

### LINKED ISSUE #1274: Sum of cubes of first N natural numbers
In C, C++, Java, Python and Go

### PR DESCRIPTION
**Pull request by dipu989**
Fixes #1274 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

#### Changes proposed in this pull request:
- Added program for sum of cubes of first N natural Numbers.

#### Languages Used:
- JavaScript

#### Files Added:
- Sum_Of_Cubes.js

### CODE REVIEW

**Comment by dipu989** [comment]
@marwahmanbir  Can you please review this?

File: Sum_Of_Cubes/Sum_Of_Cubes.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+    Sum of cubes of first N natural numbers can be calculated using the following equation
+
+    Sum = (N*(N+1)/2)^2
+    i.e 1^3 + 2^3 + 3^3 + 4^3 + ..... + N^3
+
+*/
+
+var number = window.prompt("Enter the number:","");
```
Comment (MastersAbh) [inline Sum_Of_Cubes/Sum_Of_Cubes.js]: space after ,

File: Sum_Of_Cubes/Sum_Of_Cubes.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+var n = parseInt(number);
+var res = sumOfCubes(n);
+window.alert("Sum of cubes of first "+n+" natural numbers is "+res);
```
Comment (MastersAbh) [inline Sum_Of_Cubes/Sum_Of_Cubes.js]: spaces around +

File: Sum_Of_Cubes/Sum_Of_Cubes.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+function sumOfCubes(n)
+{
+    var sum = Math.pow((n*(n+1)/2),2);
```
Comment (MastersAbh) [inline Sum_Of_Cubes/Sum_Of_Cubes.js]: spaces around operators

**Comment by dipu989** [comment]
@MastersAbh I have followed up with the changes Request.
Can you please check?

**Pull request merged**